Demos/02-extract/README.md

```diff
@@ -67,10 +67,10 @@ In this step you will create an Azure Storage account where Graph Data Connect w
 
       ![Screenshot of the Azure Storage permissions](./../../Images/azstorage-config-01.png)
 
-    1. Select the **Add role assignment** button in the navigation.
+    1. Select the **Add** button in the **Add a role assignment** block.
     1. Use the following values to find the application you previously selected to grant it the **Storage Blob Data Contributor** role, then select **Save**:
 
-        - **Role**: Storage Blob Data Contributor
+        - **Role**: Storage Blob Data Contributor (Preview)
```

Comment:
I think this role is now out of preview, so we don't need the preview tag https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#storage-blob-data-contributor